src/components/tasks/card/index.tsx

```diff
@@ -242,15 +262,15 @@ const Card: FC<Props> = ({
           <span className={classNames.cardSpecialFont}>Assignee:</span>
           <span
             className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'assignee')}
             role="button"
             tabIndex={0}
+            ref={assigneeNameRefElement}
           >
             {cardDetails.assignee}
           </span>
           <span
-            className={classNames.contributorImage}
+            className={`${classNames.contributorImage} ${shouldEdit && classNames.cursorPointer}`}
```

Comment:
Can we have a better className `cursorPointer` ?

Reply:
i think it clearly says that the `classname ` is added to make the cursor pointer when hovered, please let me know if you think otherwise

Reply:
Please refer to my above comment for the same.

Reply:
ok

Reply:
LGTM. Thanks for the change.